src/components/Modal/styles/ModalTitle.js

```diff
@@ -1,6 +1,12 @@
 import SectionTitle from 'components/SectionTitle';
 import styled from 'styled-components';
+import Anchor from 'components/Anchor';
 
 export default styled(SectionTitle)`
   padding: 0.95em 1em 0.95em 1.5em;
+
+  & > a {
```

Comment:
A stronger selector would be to pass a class name to the close button anchor and reference it here instead. This works hand-in-hand with the overrideable prop suggestion.